src/SIM/SIMoutput.C

```diff
@@ -1801,3 +1821,18 @@ bool SIMoutput::serialize (std::map<std::string,std::string>&) const
             <<"     Restart not supported for "<< this->getName() << std::endl;
   return false;
 }
+
+
+bool SIMoutput::writeAddFuncs(int iStep, int& nBlock, int idBlock, double time)
+{
+  if (myAddScalars.empty())
+    return true;
```

Comment:
This check is not needed. If empty, the loop below will handle it correctly?

Reply:
indeed.